From hartleys@visionengravers.com  Thu Aug 13 16:58:31 2009
From: "H Hartley Sweeten" <hartleys@visionengravers.com>
Date: Thu, 13 Aug 2009 13:18:02 -0400
Subject: USB: ohci-ep93xx.c: remove unused variable
To: <linux-arm-kernel@lists.arm.linux.org.uk>
Cc: "Lennert Buytenhek" <buytenh@wantstofly.org>, <linux-usb@vger.kernel.org>
Message-ID: <BD79186B4FD85F4B8E60E381CAEE190901BE9D26@mi8nycmail19.Mi8.com>


Remove unused variable in ohci-ep93xx.c.

This only shows up when CONFIG_PM is enabled.

Signed-off-by: H Hartley Sweeten <hsweeten@visionengravers.com>
Cc: Lennert Buytenhek <kernel@wantstofly.org>
Cc: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-ep93xx.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/host/ohci-ep93xx.c
+++ b/drivers/usb/host/ohci-ep93xx.c
@@ -188,7 +188,6 @@ static int ohci_hcd_ep93xx_drv_resume(st
 {
 	struct usb_hcd *hcd = platform_get_drvdata(pdev);
 	struct ohci_hcd *ohci = hcd_to_ohci(hcd);
-	int status;
 
 	if (time_before(jiffies, ohci->next_statechange))
 		msleep(5);
